src/seedcase_sprout/paths.py

```diff
@@ -8,6 +8,8 @@
 
 from pathlib import Path
 
+from seedcase_sprout.internals.create import _in_snake_case
```

Comment:
```suggestion
from seedcase_sprout.internals import _in_snake_case
```

Refer to the internals module, not the specific one.